examples/object_detection/raspberry_pi/detect.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2023 The MediaPipe Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Main scripts to run object detection."""
+
+import argparse
+import sys
+import time
+
+import cv2
+import mediapipe as mp
+
+from mediapipe.tasks import python
+from mediapipe.tasks.python import vision
+
+from utils import visualize
+
+
+def run(model: str, max_results: int, score_threshold: float, 
+        camera_id: int, width: int, height: int) -> None:
+  """Continuously run inference on images acquired from the camera.
+
+  Args:
+    model: Name of the TFLite object detection model.
+    max_results: Max of classification results.
+    score_threshold: The score threshold of detection results.
+    camera_id: The camera id to be passed to OpenCV.
+    width: The width of the frame captured from the camera.
+    height: The height of the frame captured from the camera.
+  """
+
+  # Variables to calculate FPS
+  counter, fps = 0, 0
+  start_time = time.time()
+
+  # Start capturing video input from the camera
+  cap = cv2.VideoCapture(camera_id)
```

Comment:
@PaulTR @kinaryml I believe that OpenCV can't be used to capture video stream on Bullseye because Raspberry Pi OS moved to use the libcamera stack. The workaround is to enable the legacy camera stack.
https://www.youtube.com/watch?v=E7KPSc_Xr24

Do you encounter this issue as well? If so, I think we should add that instruction to the README file for each sample. WDYT?

Reply:
I'll have to double check - I tend to have the legacy camera enabled because of my older ribbon cameras. I do think having a note about "you may need to do this" is valuable though.

Reply:
I believe OpenCV does have plans to add support for the libcamera stack. Here's the original issue:
https://github.com/opencv/opencv/issues/21653